pkg/kademlia/routing_helpers_test.go

```diff
@@ -70,145 +70,196 @@ func TestAddNode(t *testing.T) {
 	assert.NoError(t, err)
 	assert.NotNil(t, bucket)
 	cases := []struct {
-		testID string
-		node   *proto.Node
-		added  bool
-		ks     int
-		ns     int
-		nn     []int
+		testID  string
+		node    *pb.Node
+		added   bool
+		kadIDs  [][]byte
+		nodeIDs [][]string
 	}{
 		{testID: "PO: add node to unfilled kbucket",
-			node:  mockNode("PO"),
-			added: true,
-			ks:    1,
-			ns:    2,
-			nn:    []int{2},
+			node:    mockNode("PO"),
+			added:   true,
+			kadIDs:  [][]byte{[]byte{255, 255}},
+			nodeIDs: [][]string{[]string{"OO", "PO"}},
```

Comment:
you don't have to add the `[]byte` and `[]string`... so these could be:
```
kadIDs:  [][]byte{{255, 255}},
nodeIDs: [][]string{{"OO", "PO"}},
```

For all the rest as well. (you can use replace all to fix these quickly)

Reply:
ok sweet